### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by thomasaarholt** [matplotlib/ipympl#208]
There is now a lot of really nice functionality in ipympl, but it is hard to find. The in-depth documentation at ipywidgets is a really good supporting resource, but some aspects (like `display(fig.canvas)` and not `display(fig)` need to be clearly expressed.

I'm happy to write documentation for ipympl, but to do that we need some sort of framework to write in. Can someone help with this?

**Comment by martinRenou** [comment]
I totally agree and I can definitely help with this. I would suggest taking another project as starting point, like `ipyleaflet`, which uses sphinx and readthedocs.

Just copy the `docs` directory and paste it in here. Then remove almost everything so that only what's necessary remains :)

**Comment by thomasaarholt** [comment]
I'll make a PR of this, with what I can manage + a bunch of questions/topics that need some clarification.

**Comment by martinRenou** [comment]
Thanks a lot for this!

If you want to build the documentation, run `make html` from the docs directory. You would need sphinx installed:
```
conda install -c conda-forge sphinx shpinx_rtd_theme
```

**Comment by ianhi** [comment]
@thomasaarholt after #434 we now have docs (hopefully soon to be live on the web). Can we close this, or are there some things you would like to add?
